code_schemes/kakuma_s03e01_reasons.json

```diff
@@ -3,7 +3,70 @@
   "Name": "KAKUMA S03E01 Reasons",
   "Version": "0.0.0.2",
   "Codes": [
-
+    {
+      "CodeID": "code-a145bde3",
+      "CodeType": "Normal",
+      "DisplayText": "extra tuition",
```

Comment:
This is "Extra Tutoring/ Tuition" in the spreadsheet, not sure if that matters

Reply:
Was wondering whether to keep it too as they seem the same to me. Have added it back just in case.